doc/mssqlserver/doc.md

```diff
@@ -0,0 +1,145 @@
+Galleon Layers
+=========
+
+* `mssqlserver-datasource`: Provision the `MSSQLServerDS` non xa datasource. Depends on `mssqlserver-jre8-driver` layer.
+* `mssqlserver-jre8-driver`: Provision the `mssqlserver-jre8` driver.
```

Comment:
do we need to specify which JBoss Module provides the JDBC driver jar or is it superfluous?

Reply:
That would not hurt to document it. It is not supposed to change.